src/components/global/Playground/index.tsx

```diff
@@ -128,25 +128,58 @@ export default function Playground({
    * url, we post a message to each frame so that
    * dark mode can be enabled without a full page reload.
    */
-  useEffect(async () => {
+  const postDarkThemeMessage = async () => {
     if (frameiOS.current && frameMD.current) {
       await Promise.all([waitForFrame(frameiOS.current), waitForFrame(frameMD.current)]);
 
       const message = { darkMode: isDarkTheme };
       frameiOS.current.contentWindow.postMessage(message);
       frameMD.current.contentWindow.postMessage(message);
     }
+  };
+
+  const handleFrameRef = (ref: HTMLIFrameElement, frameMode: 'iOS' | 'md') => {
```

Comment:
```suggestion
  const handleFrameRef = (ref: HTMLIFrameElement, frameMode: 'ios' | 'md') => {
```

I recommend keeping the cases consistent here (similar to what we do in the framework)